### DISCUSSION TITLE
Testing

### DISCUSSION THREAD

**Discussion by timea-solid** [solid/solidos#105] [Archive] [answered]
One big category on the planning was about code improvements & team processes., especially TESTING. 
For this purpose, I created an overview of our most important repos: https://miro.com/app/board/uXjVOa2i-8Y=/?invite_link_id=313921585615

1. How do we check the test coverage of mashlib and pane-registry?
2. Is any repo missing?
3. Are there other ways to check the test coverage?

**Comment by jeff-zucker** [comment]
I'll be frank in saying I do not know much about testing or how to construct tests or how to mock various situations.  Could we maybe have a meeting devoted to knowledge-transfer on tests.  Perhaps Angelo or someone else more familiar with tests than I could lead.

**Reply by SharonStrats** [reply]
That sounds good.  I have also worked with tests quite a bit in the past. I might be able to help.

**Comment by ewingson** [comment]
@SharonStrats I' m a newbie tbh but maybe we can do a test unit on sunday ?

**Comment by timea-solid** [comment, marked as answer]
We decided to do some knowledge transfer about testing on SolidOS as a first step to improve testing.
